### PR TITLE
Created components

### PR DESCRIPTION
**Pull request by LukaBRa**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,47 +1,207 @@
...
+const toggleButton = (arg: ButtonToggleArgs) => {
+    if(arg.type == "search"){
+
+        searchFilterButtons.value = searchFilterButtons.value.map((btn) => {
+            if(btn.title === arg.title){
+                btn.active = true;
+            }else{
+                btn.active = false;
+            }
+            return btn;
```
Comment (KostaDokosic) [inline src/App.vue:53]: You can use spread operator if you want to do this type of operation.
`searchFilterButtons.value = searchFilterButtons.value.map(btn => ({ ...btn, active: btn.title === arg.title }));
`

File: src/App.vue [hunk 2]
```diff
@@ -1,47 +1,207 @@
...
+    <section class="container-fluid bg-darkgray padding-movie-list">
+
+        <div class="container">
+            <div class="movie-list">
+                <MovieCard />
+                <MovieCard />
+                <MovieCard />
+                <MovieCard />
+                <MovieCard />
+                <MovieCard />
```
Comment (KostaDokosic) [inline src/App.vue:115]: why is number of moviecards hardcoded ?

**Review by KostaDokosic — APPROVED** [review]
Approved, merge after you fix and clean code.
Overall its good.

**Comment by LukaBRa** [comment]
Dear Kosta,
Thank you for your feedback. I hardcoded MovieCard components until i reach
API calls module so i can do v-for. I will fix everything you said.

**Pull request merged**